src/odemis/driver/smaract.py

```diff
@@ -1062,6 +1067,10 @@ class MC_5DOF_DLL(CDLL):
     # events
     SA_MC_EVENT_MOVEMENT_FINISHED = 0x0001
 
+    event_name = {
+        0x0001: "MOVEMENT_FINISHED"
```

Comment:
The link is clearer if you write:
`SA_MC_EVENT_MOVEMENT_FINISHED: "MOVEMENT_FINISHED"`